src/PhpSpec/Console/Application.php

```diff
@@ -147,6 +147,13 @@ protected function loadConfigurationFile(InputInterface $input, ServiceContainer
 
                     $extension->load($container);
                 }
+            } elseif ('matchers' === $key && is_array($val)) {
+                foreach ($val as $class) {
+                    $id = strtolower(str_replace('Matcher', '', substr($class, strrpos($class, '\\') + 1)));
```

Comment:
Also we can check if given classes exist and implement `PhpSpec\Matcher\Matcher` interface.
For now, if given matcher is not `PhpSpec\Matcher\Matcher`, `phpspec` silently exists with non-zero status code.